lib/extensions/build_context_extension.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+extension BuildContextExtension on BuildContext {
+  bool get isSmScreen => MediaQuery.of(this).size.width <= 320;
```

Comment:
Also you can write it like this. 

```suggestion
  bool get isSmScreen => MediaQuery.sizeOf(this).width <= 320;
```